lib/taurus/core/util/log.py

```diff
@@ -793,6 +793,15 @@ def _format_stack(self, stack_func=inspect.stack):
                         out += '<could not find suitable string representation>'
         return out
 
+    def loggingtofile(self, msg, logging_path, *args, **kw):
+
+        file_name = logging_path + "/spock_session.log"
+
+        logging_file = open(file_name, 'a')
+        logging_file.write('\n')
+        logging_file.write(msg)
+        logging_file.close()
+
```

Comment:
it would be nice to protect this against IO errors. For example, lines 800-803 can be better rewritten as:

```
with open(file_name, 'a') as logging_file:
    logging_file.write('\n')
    logging_file.write(msg)
```